.env.dist

```diff
@@ -62,6 +62,18 @@ RALPH_BACKENDS__DATA__ES__TEST_HOSTS=http://elasticsearch:9200
 RALPH_BACKENDS__DATA__ES__TEST_INDEX=test-index-foo
 RALPH_BACKENDS__DATA__ES__TEST_FORWARDING_INDEX=test-index-foo-2
 
+# ES lrs backend
+
+RALPH_BACKENDS__LRS__ES__HOSTS=http://elasticsearch:9200
+RALPH_BACKENDS__LRS__ES__DEFAULT_INDEX=statements
+# RALPH_BACKENDS__LRS__ES__ALLOW_YELLOW_STATUS=False
+# RALPH_BACKENDS__LRS__ES__CLIENT_OPTIONS__ca_certs=False
+# RALPH_BACKENDS__LRS__ES__CLIENT_OPTIONS__verify_certs=False
+# RALPH_BACKENDS__LRS__ES__DEFAULT_CHUNK_SIZE=500
+# RALPH_BACKENDS__LRS__ES__LOCALE_ENCODING=utf8
+# RALPH_BACKENDS__LRS__ES__POINT_IN_TIME_KEEP_ALIVE=1m
+# RALPH_BACKENDS__LRS__ES__REFRESH_AFTER_WRITE=False
+
```

Comment:
Good point) Indeed, it would be better to avoid repetition. However, it might be helpful to indicate that configuration is not shared between `lrs` and `data` backends.
Replaced this part with:
```
# ES lrs backend

# Same options as for the ES data backend, however they are prefixed with
# RALPH_BACKENDS__LRS__ES__ instead. Example:
# RALPH_BACKENDS__LRS__ES__HOSTS=http://elasticsearch:9200
```
WDYT?

Reply:
I like it!